x-pack/test/functional/page_objects/api_keys_page.ts

```diff
@@ -26,5 +27,66 @@ export function ApiKeysPageProvider({ getService }: FtrProviderContext) {
     async apiKeysPermissionDeniedMessage() {
       return await testSubjects.getVisibleText('apiKeysPermissionDeniedMessage');
     },
+
+    async clickOnCreateApiKey() {
+      return await (await find.byCssSelector('a[data-test-subj="createApiKey"]')).click();
+    },
+
+    async setApiKeyName(apiKeyName: string) {
+      return await (
+        await find.byCssSelector('input[data-test-subj="apiKeyName"]')
```

Comment:
```
testSubjects.setValue('apiKeyName', value);
```

(Here's the test subject service definition with all available methods `test/functional/services/common/test_subjects.ts`)